File: problems/problems_3568/solution.ts
function minMoves(classroom: string[], energy: number): number {
    
};

export function Solve(inputJsonElement: string): any {
	const inputValues: string[] = inputJsonElement.split("\n");
	const classroom: string[] = JSON.parse(inputValues[0]);
	const energy: number = JSON.parse(inputValues[1]);
	return minMoves(classroom, energy);
}

File: problems/problems_3568/solution.go
package problem3568

import (
	"encoding/json"
	"log"
	"strings"
)

func minMoves(classroom []string, energy int) int {
    
}

func Solve(inputJsonValues string) any {
	inputValues := strings.Split(inputJsonValues, "\n")
	var classroom []string
	var energy int

	if err := json.Unmarshal([]byte(inputValues[0]), &classroom); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &energy); err != nil {
		log.Fatal(err)
	}

	return minMoves(classroom, energy)
}
